app/code/Magento/Ui/Test/Mftf/Test/AdminDisappearErrorValidateMessageFieldTest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<!-- Test XML Example -->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminDisappearErrorValidateMessageFieldTest">
```

Comment:
I would rather move this test under Magento_Bundle module, as the test is more related to Bundle product. What do you think?

Reply:
We change UI component and I think a test must be in module Ui. The test scenario is related to an issue. Maybe there are other places where the issue is reproduced but I didn't find them.

Reply:
Generally, a non written rule says that we shouldn't be using ActionGroups/Entities, etc. for modules that we don't have in dependencies. But seeing that Ui > Sales > Bundle, most likely we should be fine here.